### DISCUSSION TITLE
Wix v5 has incorrect screen reader narration order by default

### DISCUSSION THREAD

**Discussion by jmjaffe37** [wixtoolset/issues#8179] [Questions]
The default narration order for the first screen of the default setup wizard (with MSIs generated with wix) seems to be:

1. Title bar
1. Next button (says 'Alt and n' after and adds 'disabled' if the button is disabled)
1. Cancel button
1. Wix dialog Image file name
1. Back button (says 'Alt and b' after and adds 'disabled' if the button is disabled)
1. Setup wizard description
1. Setup wizard title

The narration order seems to be wokny for most screens by default for most/all screens. It seems to always read out the filenames of images included on the current MSI page as well. Tested default pages:
1. welcome dialog
1. change, repair, remove dialog
1. finish screen
1. end-user license agreement (this order is fine, but it still reads out the filename of the logo/image, and it does not read the dialog title 'End-User License Agreement', nor doe it read the description of that title)
1. CustomDialog (same issues as end-user license agreement page -- title not read, png filename read)
1. Custom Setup dialog only reads title bar and first element selected in tree
1. Installation screen reads everything but puts the dialog title and description last. Also, this still reads out the filename of the log

link to well-known download page with MSI built with wix v4 that has this problem (downloads page of "Accessibility Insights" wesbite):
https://accessibilityinsights.io/downloads/

I am also experiencing this problem in my own testing with wix v5

**Comment by jmjaffe37** [comment]
This is related to this discussion question I asked last week which does not have an answer yet:
https://github.com/orgs/wixtoolset/discussions/8160

**Comment by barnson** [comment]
There are a number of open accessibility bugs and order is a common one: https://github.com/wixtoolset/issues/issues?q=is%3Aopen+is%3Aissue+label%3Aaccessibility
